homeassistant/components/sensor/nederlandse_spoorwegen.py

```diff
@@ -135,13 +135,21 @@ def device_state_attributes(self):
             'departure_delay':
                 self._trips[0].departure_time_planned !=
                 self._trips[0].departure_time_actual,
+            'departure_platform':
+                self._trips[0].trip_parts[0].stops[0].platform,
```

Comment:
Did you test these changes? Your PR seems to have been made via GitHub…

Reply:
Yeah, I did. I’m fairly new to both Github and Python, so I actually developed and tested this on my local home assistant installation (it worked as espected). Then I wanted to publish my changes on the forums but decided the Github option would be the better option ;)

What’s the best way? Still learning...

Reply:
Your approach works (as this PR shows) but it's not great for tracking changes or migrating changes to a branch. Best way is to set up a development environment so that you can run all checks locally: https://home-assistant.io/developers/development_environment/

Reply:
Thanks much. If I ever get to submit another idea / improvement, I will :)